<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="X3552" ignored="0" name="Test" tests="1" failures="1" timestamp="07 Jan 2020 22:50:31 GMT" time="60.279" errors="0">
  <testcase name="main" time="60.255" classname="AMOS_Regression.AMOSActivation">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at AMOS_Regression.AMOSActivation.main(AMOSActivation.java:71)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- main -->
</testsuite> <!-- Test -->
